packages/components/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,194 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconDotsHorizontal } from '@frontify/fondue-icons';
+import * as RadixTabs from '@radix-ui/react-tabs';
+import { createContext, forwardRef, useContext, useEffect, useRef, type ForwardedRef, type ReactNode } from 'react';
+
+import { Button } from '../Button/Button';
+import { Dropdown } from '../Dropdown/Dropdown';
+
+import { useTabTriggers } from './hooks/useTabTriggers';
+import styles from './styles/tabs.module.scss';
+import { type TabTrigger } from './types';
+
+import { useControllableState } from '#/hooks/useControllableState';
+
+export type TabsRootProps = {
+    id?: string;
+    children: ReactNode;
+    /**
+     * The default active tab
+     * Used for uncontrolled components
+     */
+    defaultActiveTab?: string;
+    /**
+     * The controlled value of the active tab
+     */
+    activeTab?: string;
+    /**
+     * The height of the tabs
+     */
+    size?: 'default' | 'large';
```

Comment:
No other component has `default` as size let's align it

Reply:
fixed